# environment cofig
actions:
  max_speed: 3.0
  max_turn: 0.434 # maximum angular velocity
  min_speed: 0.0
  min_turn: -0.434
